app/controllers/api/files_controller.rb

```diff
@@ -0,0 +1,37 @@
+class API::FilesController < APIController
+  PUBLIC_FILES = Rails.public_path.join('files').children
+
+  def show
+    if file
+      send_file file, disposition: :inline
+    else
+      render json: { error: 'Fichier non trouvé' }, status: :not_found
```

Comment:
```suggestion
      render plain: 'Fichier non trouvé', status: :not_found
```

aucune raison de rendre un json ici